js/apps/admin-ui/src/user/EditUser.tsx

```diff
@@ -185,6 +194,24 @@ export default function EditUser() {
         titleKey={user.username!}
         className="kc-username-view-header"
         divider={false}
+        badges={
+          lightweightUser
+            ? [
+                {
+                  text: (
+                    <Tooltip content={t("transientUserTooltip")}>
+                      <Label
+                        data-testid={"user-details-label-transient-user"}
```

Comment:
String literals can be passed directly.
```suggestion
                        data-testid="user-details-label-transient-user"
```